.github/workflows/ci.yml

```
@@ -104,4 +104,4 @@ jobs:
       # Test probably delete this
       - name: Smoke Test
         shell: bash -l {0}
-        run: python run_stac.py
\ No newline at end of file
+        run: python run_stac.py --config-name config_synth_data.yaml
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)